### ISSUE TITLE
Horizontal scrolling no more working since release 9.x

### ISSUE THREAD

**Issue by muecke87** [wicketstuff/core#737]
With https://github.com/wicketstuff/core/commit/26ca873f5118a122df16988837839e1e1cc25e6a browser detection was removed.
Unfortunately the following row in ` script-jq.js` was also removed:

```
if (this.isIE || this.isGecko) {
  			bodyContainer1.style.width = topContainer.offsetWidth + "px";
  		}  	 
```

This alleged "IE" or "GECKO" fix seems also to be necessary for "all" well known browsers as well (tested with chrome, firefox and safari). Without that code, horizontal scrolling is broken. So the fix would be a oneliner on line **542**:

`  			bodyContainer1.style.width = topContainer.offsetWidth + "px";
`

Will this be fixed? I monkey patched it in our project but that's not the proper way ;-)

**Comment by martin-g** [comment]
Please send a Pull Request!

**Comment by muecke87** [comment]
I tried but it seems that I don't have permission to push: Permission to wicketstuff/core.git denied to ...

**Comment by martin-g** [comment]
that's normal - you are not a member of wicketstuff team at Github.
you should push to your fork and create a Pull Request - https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/creating-an-issue-or-pull-request

**Comment by muecke87** [comment]
okay, will do (although we do not really need/want a fork).
sorry I am new to contributing on github therefore two questions:
* I'll fix this in the master of the fork
* Any ideas when this fix will be released from the original repo?

Am Mi., 7. Dez. 2022 um 10:21 Uhr schrieb Martin Grigorov <
***@***.***>:

**Comment by muecke87** [comment]
okay will do, although we don't really need/want a fork.
I am new at contributing therefore two questions:
* Should I fix this issue on the fork's master?
* Any ideas when and in which release this might be published in wicketstuff core?

**Comment by martin-g** [comment]
* you do need to fork if you want to contribute!
* you can make the fix in any branch in your fork. It is recommended to use a branch! When creating a PR just make sure it is against `master` branch of wicketstuff/core.
* WicketStuff core is release few days after Apache Wicket 9.x is released. So, it depends when Wicket 9.13.0 will be released.

**Comment by muecke87** [comment]
All clear, thx.
I want a bug fixed, not really contribute, but fine ;-)

**Comment by martin-g** [comment]
You can wait someone else to do it :-)
It might take days or years though ...

**Comment by muecke87** [comment]
thx for approving. not shure about the process: do you close the issue or I? or will it be closed automatically and only when new version was released?

**Comment by martin-g** [comment]
Fixed with #738
